src/pages/Home/Logo/Logo.tsx

```diff
@@ -0,0 +1,63 @@
+type PropsTitleBar = {
+	textcolor: string | undefined | null;
+};
+
+export default function Logo(props: PropsTitleBar) {
+	const fillColor = props.textcolor === null ? '#F9F9F9' : props.textcolor;
```

Comment:
small suggestion here:
```suggestion
	const fillColor = props.textcolor || '#F9F9F9';
```